src/Components/Gnb.tsx

```diff
@@ -1,37 +1,42 @@
 import { useNavigate } from 'react-router-dom'
 import styled from '@emotion/styled'
 import logoIcon from '../Asset/logo.svg'
+import { useSelector } from 'react-redux'
+import { RootState } from '../state/store'
 import { palette, typography } from '../styles/theme'
 import { useCallback, useState } from 'react'
-import { useSelector } from 'react-redux'
-import type { RootState } from '../state/store'
 
-export const Gnb = (props: { loginState: boolean }) => {
+interface Props {
+  loginState: boolean
+}
+
+export const Gnb = (props: Props) => {
   const navigate = useNavigate()
+  const cid = useSelector((state: RootState) => state.userInfo.classes[0])
```

Comment:
perhaps later, I think we should change this first class `classes[0]` logic so that students can choose their class

Reply:
> perhaps later, I think we should change this first class `classes[0]` logic so that students can choose their class

I think that's a good idea :)